scripts/zpool

```diff
@@ -53,7 +53,22 @@ zpool_monitor() {
     # exit immediately if configuration is not valid
     zpool_validate_all || exit $?
 
-    ocf_run -q -info zpool status $OCF_RESKEY_pool
+    if [ -z "$OCF_RESKEY_pool" ] ; then
```

Comment:
Please remove this check.  OCF_RESKEY_pool is a _required_ option in this agent.  Even if we couldn't assume that it is provided, "false", and therefore OCF_NOT_RUNNING would not be the correct exit code.  It would probably be something like OCF_ERR_CONFIGURED.

In the future if we have some sanity checking that we want to do on the input parameters, those should go into the zpool_validate_all() function.